### DISCUSSION TITLE
成功企業AI

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#595] [Show and tell]
https://www.sas.com/zh_tw/insights/articles/analytics/sandbox.html?utm_source=google&utm_medium=cpc&utm_campaign=sas-aai-ap&gclid=Cj0KCQjwpv2TBhDoARIsALBnVnmLg9lVmka474K2AJC54rH5wm1qTBJNhzj0EMPltF6VO2ByG6dBQBgaAuzwEALw_wcB&gclsrc=aw.ds
文章提到，至今成功的企業AI最重要的點在於分析，進而去做到決定最佳的商業決策。最後文章提到，最理想化的是能做到各個AI間的技術交流，讓AI這塊能發展得更美好。

**Comment by handsomeniko** [comment]
希望AI能讓生活更好

**Comment by Jay-722** [comment]
分析不同情況以達到最佳決策，互相交流能使得技術更近一步

**Comment by Sazmafuji** [comment]
AI未來到底會往哪方向發展阿

**Comment by yucheng1028** [comment]
真的好期待

**Comment by d26488399** [comment]
真的值得期待

**Comment by cordydeng** [comment]
未來AI不知道還能擴展到什麼方面，只希望不要把我們取代掉

**Comment by 110409538** [comment]
AI的未來發展值得期待

**Comment by HsuChen88** [comment]
會不會未來管理階層都被AI取代

**Comment by ghost** [comment]
不會到完全取代 但人也相對提升自身優勢就是了
